src/commands/scanner/run.ts

```diff
@@ -79,14 +79,11 @@ export default class Run extends ScannerCommand {
     format: flags.enum({
       char: 'f',
       description: messages.getMessage('flags.formatDescription'),
-      options: [OUTPUT_FORMAT.XML, OUTPUT_FORMAT.JUNIT, OUTPUT_FORMAT.CSV, OUTPUT_FORMAT.TABLE],
-      default: OUTPUT_FORMAT.TABLE,
-      exclusive: ['outfile']
```

Comment:
If you're getting rid of the exclusivity, surely you still need some kind of validation, right?

Reply:
What additional validation do you think is needed that the exclusivity formerly provided?

Reply:
I imagine you'd want to disallow `--format csv --outfile something.xml` or `--format xml --outfile something.csv`, right? There's probably a few other combinations that are invalid.

Reply:
I think I would lean toward liberty on that front.  If someone really wants to write xml to a .csv file, they may have a weird but good reason for that.  It doesn't introduce any risk that I can see.  Your OS lets you rename any file type to any extension.

Reply:
Fair enough, but perhaps it's worth logging a warning?

Reply:
Cool.  Will add that.